apps/regression/base.hu.i18n

```diff
@@ -1,21 +1,20 @@
 RegressionApp = "Regresszió"
 RegressionAppCapital = "REGRESSZIÓ"
 Regression = "Regresszió"
-Reg = "reg"
 MeanDot = "Középérték"
 RegressionCurve = "Regressziós görbe"
-XPrediction = "Jóslás megadva X"
+XPrediction = "Jóslás X megadva"
 YPrediction = "Jóslás Y megadva"
 ValueNotReachedByRegression = "Az ablakban az érték még nem volt elérve"
 NumberOfDots = "Pontok száma"
 Covariance = "Kovariancia"
 Linear = "Lineáris"
-Proportional = "Proportional"
+Proportional = "Általános"
 Quadratic = "Másodfokú"
-Cubic = "Kocka"
+Cubic = "Kockás"
 Quartic = "Kvartikus"
 Logarithmic = "Logaritmikus"
 Power = "Teljesítmény"
 Trigonometrical = "Trigonometrikus"
 Logistic = "Logisztikai"
-DataNotSuitableForRegression = " Az adat nem megfelelö ehhez a regressziós modellhez"
+DataNotSuitableForRegression = "Azek az adatok nem megfelelö ehhez a regressziós modellhez"

```

Comment:
"Ezek az adatok nem megfelelőek ehhez a regressziós modellhez" <-- this is correct